### ISSUE TITLE
SQLformat error

### ISSUE THREAD

**Issue by alumni** [Glavin001/atom-beautify#2075]
with the latest version of SQLformat and atom-beautify, I keep getting this error:

```
Error: usage: sqlformat  [OPTIONS] FILE, ...
sqlformat: error: argument -r/--reindent: ignored explicit argument 'true'
    at file:///~/.atom/packages/atom-beautify/src/beautifiers/executable.coffee:182:23
    at tryCatcher (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:693:18)
    at Async._drainQueue (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:143:10)
    at Async.drainQueues (~\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:17:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by JFFI** [comment]
I'm getting the same error. 
I tried to set the Reindent option to false but it doesn't fix it, instead it just outputs the same message with the argument set to 'false'. My guess is that it may have something to do with the way this argument is passed to sqlformat.
```
usage: sqlformat  [OPTIONS] FILE, ...
sqlformat: error: argument -r/--reindent: ignored explicit argument 'false'
```

**Comment by thehongwu** [comment]
same here, it happens recently, is it a error from sql parse package or auto-beautify?

**Comment by Jonnokc** [comment]
+1

`usage: sqlformat [OPTIONS] FILE, ... sqlformat: error: argument -r/--reindent: ignored explicit argument 'false'`

Version info: 
sqlformat: 0.2.4
atom-beautify: 0.32.0
atom: 1.24.0x64

**Comment by szeck87** [comment]
https://github.com/Glavin001/atom-beautify/pull/1926

**Comment by szeck87** [comment]
@faheel This looks related to #1926.

**Comment by szeck87** [comment]
Once https://github.com/Glavin001/atom-beautify/pull/2064 is published, this will be fixed.

**Comment by Glavin001** [comment]
Published to v0.32.2
